SDL/SDLMain.cpp

```diff
@@ -106,24 +108,25 @@ static void InitSDLAudioDevice(const std::string &name = "") {
 
 	audioDev = 0;
 	if (!startDevice.empty()) {
-		audioDev = SDL_OpenAudioDevice(startDevice.c_str(), 0, &fmt, &ret_fmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
+		audioDev = SDL_OpenAudioDevice(startDevice.c_str(), 0, &fmt, &g_retFmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
 		if (audioDev <= 0) {
 			WLOG("Failed to open audio device: %s", startDevice.c_str());
 		}
 	}
 	if (audioDev <= 0) {
-		audioDev = SDL_OpenAudioDevice(nullptr, 0, &fmt, &ret_fmt, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE);
+		ILOG("SDL: Trying a different sample rate");
```

Comment:
This is actually trying a difference device, they both allow a freq change.

-[Unknown]

Reply:
Oh right, was reading carelessly, apparently.